### PR TITLE
Clarify feed format when using preprocessed operations

### PR DESCRIPTION
**Pull request by jobergum**
Yes. I confirm that this contribution is made under the terms of the license found in the root directory of this repository's source tree and that I have the authority necessary to make this contribution on behalf of its copyright owner.

### CODE REVIEW

File: documentation/feed-using-hadoop-pig-oozie.html [hunk 1]
```diff
@@ -54,20 +54,26 @@ <h2 id="pig">Pig</h2>
...
+<p>
 To run the above Pig script, specify <a href="#vespa.feed.endpoint">endpoint</a>.
-The endpoint is whatever Vespa endpoint to feed to.
-<code>VespaStorage</code> supports feeding to multiple endpoints.
-To specify multiple endpoints, separate them by commas.
+The endpoint is the Vespa endpoint (hostname) to feed to.
 Importantly, the endpoint should be defined as it is for the
 <a href="vespa-http-client.html">Vespa HTTP client</a>,
-which means the endpoint host name, without 'http://' and port number.
+which means the endpoint is equal the hostname, without schema ('http(s)://' and without port number.
```
Comment (lesters) [inline documentation/feed-using-hadoop-pig-oozie.html]: is equal *to* the hostname ?

File: documentation/feed-using-hadoop-pig-oozie.html [hunk 2]
```diff
@@ -90,6 +96,13 @@ <h3 id="feeding-preconstructed-operations">Feeding preconstructed operations</h3
 -- Store into Vespa
 STORE data INTO '$ENDPOINT' USING VespaStorage();
 </pre>
+The source, e.g 'feed-dump.json' needs to contain exactly one valid JSON document operation per line, e.g 
+<pre>
+{"put": "id:namespace:music::0","fields": {"title":"foo"}}
+{"put": "id:namespace:music::1","fields": {"title":"foo"}}
+{"update": "id:namespace:music::2","fields": { "title": { "assign":"bar"}}}
+</pre>
+Hence, pretty-printed JSON will not work.
```
Comment (lesters) [inline documentation/feed-using-hadoop-pig-oozie.html]: Actually, there is a way to load multi-line json: https://github.com/vespa-engine/vespa/blob/master/vespa-hadoop/src/test/pig/feed_multiline_operations.pig

File: documentation/feed-using-hadoop-pig-oozie.html [hunk 3]
```diff
@@ -259,7 +272,8 @@ <h3 id="feeding-status">Feeding status</h3>
 ...
 </pre>
 Here, Pig reports that is Successfully stored 100001 records,
-but <code>VespaStorage</code> skipped one probably due to a formatting error.
+but <code>VespaStorage</code> skipped one probably due to a formatting error and only 100000 operations was sent. The skipped counter is incremented if
```
Comment (lesters) [inline documentation/feed-using-hadoop-pig-oozie.html]: Above here: ... that *it* succesfully stored ...

File: documentation/feed-using-hadoop-pig-oozie.html (hunk 2 above)
Comment (jobergum) [reply documentation/feed-using-hadoop-pig-oozie.html]: Great, added example of that as well.

**Pull request merged**